TECHMANIA/Assets/Scripts/Components/Main Menu/SelectSkinPanel.cs

```diff
@@ -30,11 +30,25 @@ public class SelectSkinPanel : MonoBehaviour
     private List<GameObject> vfxInstances;
 
     private void InitializeDropdown(TMP_Dropdown dropdown,
-        string skinFolder, string currentSkinName)
+        string skinFolder, string skinSaFolder, string currentSkinName)
     {
         dropdown.options.Clear();
         int value = 0, index = 0;
         bool foundOption = false;
+        foreach (string folder in
+            Directory.EnumerateDirectories(skinSaFolder))
+        {
+            // folder does not end in directory separator.
+            string skinName = Path.GetFileName(folder);
+            if (skinName == currentSkinName)
+            {
+                value = index;
+                foundOption = true;
+            }
+            index++;
+            dropdown.options.Add(new TMP_Dropdown.OptionData(skinName));
```

Comment:
Due to how Paths.GetNoteSkinFolder etc. are implemented, if there are skins of the same name in both skin folder and streaming assets skin folder, the latter becomes inaccessible.
To match that behavior, we can process the skin folder first and streaming assets second, and skip a skin if a skin of the same name already exists in the dropdown.